jetty-core/jetty-tests/jetty-test-client-transports/src/test/java/org/eclipse/jetty/test/client/transport/ConnectionPoolTest.java

```diff
@@ -54,15 +55,25 @@ public void testPreCreateConnections(Transport transport) throws Exception
         destination.getConnectionPool().preCreateConnections(client.getMaxConnectionsPerDestination())
             .get(5, TimeUnit.SECONDS);
 
-        // Verify that connections have been created.
-        List<Connection> connections = switch (transport)
+        // Verify that server connections have been created.
+        await().atMost(5, TimeUnit.SECONDS).untilAsserted(() ->
         {
-            case HTTP, HTTPS -> serverConnections.filter(HttpConnection.class);
-            case H2C, H2 -> serverConnections.filter(HTTP2ServerConnection.class);
-            case H3 -> serverConnections.filter(ServerQuicConnection.class);
-            case FCGI -> serverConnections.filter(ServerFCGIConnection.class);
-        };
-        assertThat(connections, not(empty()));
+            switch (transport)
+            {
+                case HTTP, HTTPS:
+                    assertThat(serverConnections.filter(HttpConnection.class).size(), is(maxConnectionsPerDestination));
+                    break;
```

Comment:
Boo! Using old-style `switch` with `:` and `break` 🤣 
Please use new style.

Reply:
Done.